includes/class-sensei.php

```diff
@@ -568,6 +569,11 @@ public function initialize_global_objects() {
 
 		// Cron for periodically cleaning guest user related data.
 		Sensei_Temporary_User_Cleaner::instance()->init();
+
+		$email_customization_enabled = $this->feature_flags->is_enabled( 'email_customization' );
```

Comment:
Can we have a test to cover this checking, it is simple, but it helps us ensure we are not leaking features that are not ready yet.

Reply:
Tests added here: https://github.com/Automattic/sensei/pull/6487/commits/40ef52a98e5afa7eb7725c6c965a02454b426fe8

Reply:
Oh, I was trying to say, the test should cover if the feature flag is off, the post type should not be registered.